src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write your code here
+const allSpans = [...document.getElementsByClassName('population')];
+let totalValue = 0;
+
+for (let i = 0; i < allSpans.length; i++) {
+  const num = +allSpans[i].textContent.split(',').join('');
+
+  totalValue += num;
+}
+
+const averageValue = totalValue / allSpans.length;
```

Comment:
Ensure that 'allSpans.length' is not zero to avoid division by zero, which would result in 'Infinity'. Consider adding a check before calculating the average.